docs/team/bostoncheong.adoc

```diff
@@ -3,42 +3,62 @@
+== Project Overview : Battleship
+'Battleship' is a desktop game application based on the classic Battleship guessing game. Borrowing the same game rules,
+the game aims to recreate the adrenaline-filled experience by pitting the player against an intelligent computerized enemy.
+However, instead of using a pencil and paper, players will be introduced with the command-line interface which they
+will use to play the game.
 
-Battleship is a desktop game application based on the classic Battleship game.
-The user interacts with it using a CLI and it has a graphical user interface(GUI) created using JavaFX.
-It is written in Java, and has about 10 kLoC.
+My role was to design and implement a method to track and monitor the progress of the game. I managed to achieve this by
+introducing the 'Statistics' feature of Battleships. By allowing the player to view their gameplay data, performance and
+ score at any time during the game, players are able to plan their moves strategically.
 
-My role was to design and write the code for the `stats` feature of the game.
-The following sections illustrate the feature in more detail, as well as the relevant sections I have added to the user and developer guides in relation to this feature.
+The following sections will illustrate the feature and its implementation in more detail.
 
 == Summary of contributions
 
-* *Major enhancement*: added *statistic feature of the game*
-** *What it does*: The player can view information of their gameplay data any time during the duration of the game. This is enabled through a pop-up window that will display the relevant player data so far.
-** *Justification*: The statistics feature is crucial to user experience in the game as it allows players to track and keep up with the progress of the game. It is also fundamental as a basis for deploying strategies in the gameplay.
-** Highlights: [placeholder]
-** Credits: [placeholder]
+* *Major enhancement*: added the *ability to view the game score data at any juncture of the game*.
+** What it does: By entering the `stats` command, the player will be given a pop-up display that contains information on
+the number of successful hits, number of misses, number of enemy ships destroyed and even the player's current shooting
+accuracy.
+** Justification: This feature is crucial to user experience in the game as it allows players to track and keep up with the progress of the game. It is also fundamental as a basis for deploying strategies in the gameplay.
+
+* *Code contributed*: [https://github.com[Functional code]] [https://github.com[Test code]] _{give links to collated code files}_
+
+* *Minor enhancement*: added the *ability to compare the game scores between two games.*
+** What it does: Upon winning a game, the game will automatically save the current game statistics into the storage folder
+of the application. At the same time, the game will perform a comparison with the previous game statistics data (if any)
```

Comment:
Perhaps could phrase it as "At the same time, if there are previous game statistics, the game will make a comparison."